wgengine/netstack/netstack.go

```diff
@@ -1478,12 +1478,12 @@ func (ns *Impl) forwardTCP(getClient func(...tcpip.SettableSocketOption) *gonet.
 		ns.logf("netstack: could not connect to local backend server at %s: %v", dialAddr.String(), err)
 		return
 	}
-	defer backend.Close()
```

Comment:
I think this is still needed. `CloseRead` + `CloseWrite` shuts down both directions of communication but does not fully close the fd. Also the non-`net.TCPConn` path still needs a close somewhere.

Reply:
I wasn't sure if a double-`Close()` would be a problem, guess I erred too much on the other side